cpu/stm32/stmclk/stmclk_l4wx.c

```diff
@@ -347,9 +350,10 @@
 #define CLOCK48MHZ_SELECT           (0)
 #endif
 
-/* periph_hwrng and periph_usbdev require a 48MHz clock source */
+/* Only periph_hwrng requires 48MHz for the moment */
```

Comment:
Looks like a rebase issue, the initial comment was OK.

Reply:
Indeed I had to manually fix this part yesterday

Reply:
Fixed and squashed.